pkg/utils/pod.go

```diff
@@ -140,33 +140,59 @@ func CountReadyPods(podList *v1.PodList) (int64, error) {
 	return int64(readyPodCount), nil
 }
 
-// FilterReadyPods filters and returns a list of pods that have a valid PodIP.
-func FilterReadyPods(pods map[string]*v1.Pod) []*v1.Pod {
```

Comment:
Seems `FilterReadyPods` has been deleted? do we have any other references?

Reply:
Ok. If we do not have other caller, that's ok. If we do, we may need to keep a copy. but seems we do not have it